system/assets/hypha-image-uploader.js

```diff
@@ -0,0 +1,55 @@
+$(function() {
+	$('[data-image-uploader]').each(function () {
+		// get input
+		var $input = $(this);
+		console.log($input.data('image-uploader-src'));
+
+		var src = $input.data('image-uploader-src');
```

Comment:
Maybe this line should be moved way further down, since the `src` var is only used there (and now it might be confusing with the `src` parameter to `createPreviewImage()`.

Reply:
fixed